src/connector/src/source/cdc/external/mysql.rs

```diff
@@ -112,11 +113,51 @@ impl MySqlExternalTable {
             let data_type = mysql_type_to_rw_type(&col.col_type)?;
             // column name in mysql is case-insensitive, convert to lowercase
             let col_name = col.name.to_lowercase();
-            column_descs.push(ColumnDesc::named(
-                col_name.clone(),
-                ColumnId::placeholder(),
-                data_type,
-            ));
+            let column_desc = if let Some(default) = col.default {
+                let snapshot_value = match default {
+                    ColumnDefault::Null => None,
+                    ColumnDefault::Int(val) => match data_type {
+                        DataType::Int16 => Some(ScalarImpl::Int16(val as _)),
+                        DataType::Int32 => Some(ScalarImpl::Int32(val as _)),
+                        DataType::Int64 => Some(ScalarImpl::Int64(val)),
+                        _ => {
+                            unreachable!("unexpected default value type for integer column")
+                        }
```

Comment:
Since the `default` and `data_type` are from external systems, shall we return an error instead of panicking here?

Reply:
sounds ok to me.